lib/controller/index.js

```diff
@@ -0,0 +1,15 @@
+const program = require('caporal');
+
+const init = require('./init');
```

Comment:
Based on our reading of existing literature on ES6 ([exploring ES6](http://exploringjs.com/es6/)), we evolved the following template structure for a ES6 module.
```
imports
private functions (often async / await to make them asynchronous)
private classes
public classes
module.export of public classes
```
please see an example file named [status.js](https://github.com/AutolabJS/AutolabJS/blob/master/load_balancer/status.js) from AutolabJS project. 

I see the proposed structure to be more readable. If you don't have any objections to the given template, please adopt the same for this project as well.

One valid objection could be that a class is a extra paper work for code in input/, controller/ and output/ modules. We could get away with having a function there. Still, the other modules like model/ demand a class-based approach. 

In summary, the input/, controller/ and output/ modules can have the following structure.
```
imports
private functions (often async / await to make them asynchronous)
public functions
module.export of public functions defined earlier
```
Let's use the class-based template for code in model/ module.